qemu/tests/timedrift_with_reboot.py

```diff
@@ -30,6 +31,11 @@ def run(test, params, env):
     timeout = int(params.get("login_timeout", 360))
     session = vm.wait_for_login(timeout=timeout)
 
+    if params.get("os_type") == 'linux':
+        utils_time.sync_timezone_linux(session=session)
+    elif params.get("os_type") == 'windows':
+        utils_time.sync_timezone_windows(vm)
```

Comment:
You currently have sync_timezone_win() in utils_time. Otherwise looks good to me.